engine/src/flutter/shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -1562,10 +1562,13 @@ void PlatformViewAndroidJNIImpl::FlutterViewOnDisplayPlatformView(
       mutators_stack.Begin();
   while (iter != mutators_stack.End()) {
     switch ((*iter)->GetType()) {
-      case kTransform: {
-        const SkMatrix& matrix = (*iter)->GetMatrix();
-        SkScalar matrix_array[9];
-        matrix.get9(matrix_array);
+      case MutatorType::kTransform: {
```

Comment:
Note to self, investigate if we need to drop to a 3x3 here or if we could use a full 4x4

Reply:
The Android class being populated was a 3x3-based class. If they have a more accurate class we can consider adopting that, in a separate issue.

Reply:
Filed as https://github.com/flutter/flutter/issues/164596